src/transformState.js

```diff
@@ -6,6 +6,27 @@
  */
 function transformState(state, actions) {
   // write code here
+  for (const key of actions) {
+    switch (key.type) {
+      case 'addProperties':
+        Object.assign(state, key.extraData);
+        break;
+
+      case 'removeProperties':
+        for (const remove of key.keysToRemove) {
+          if (state.hasOwnProperty(remove)) {
```

Comment:
if we try to access non-existing property, we won't get any error, and deleting undefined one won't cause anything to our object